src/generated/resources/openapi.yaml

```diff
@@ -183,6 +183,32 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/SidecarAccessToken"
+  /gateway/v1/preferences:
+    get:
+      tags:
+      - Preferences Resource
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Preferences"
+    put:
+      tags:
+      - Preferences Resource
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Preferences"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Preferences"
```

Comment:
We should add the 400 response here, too, via annotations [like we do in the `ConnectionsResource`](https://github.com/confluentinc/ide-sidecar/blob/0e27f1c878a581e3c4f7491e5a3a6b57500ca87b/src/main/java/io/confluent/idesidecar/restapi/resources/ConnectionsResource.java#L83-L112).

Reply:
Great catch! I'll address this in a follow-up PR.